doc/install_mne_python.rst

```diff
@@ -5,87 +5,72 @@
 Installing MNE-Python
 ---------------------
 
-There are many possible ways to install a Python interpreter and MNE.
-Here we provide guidance for the simplest, most well tested solution.
+There are many ways to install a Python interpreter and MNE. Here we show a simple well tested solution.
 
-1. Get a Python interpreter
-###########################
+1. Get Python
+#############
 
-* We recommend the Anaconda Python 3+ distribution.
-  Follow `their installation instructions <http://docs.continuum.io/anaconda/install>`_.
-  When you are done, you should see some variant of this in a terminal:
+We recommend the `Anaconda distribution <https://www.anaconda.com/distribution/>`_.
+Follow the `installation instructions <http://docs.continuum.io/anaconda/install>`_.
+When you are done, you should see a similar output if you type the following command in a terminal:
 
-  .. code-block:: console
+.. code-block:: console
+
+    $ conda --version && python --version
+    conda 4.5.4
+    Python 3.6.5 :: Anaconda, Inc.
 
-      $ conda --version && python --version
-      conda 4.4.10
-      Python 3.6.4 :: Continuum Analytics, Inc.
+If you get an error message, consult the Anaconda documentation and search for Anaconda install
+tips (`Stack Overflow <https://stackoverflow.com/>`_ results are often helpful).
 
-  If it doesn't, something went wrong.
-  Look through the Anaconda documentation and Google Anaconda install
-  tips (StackExchange results are often helpful).
+.. note:: Note that MNE-Python 0.17 will be the last release to support Python 2. From MNE-Python 0.18, only Python 3 will be supported.
```

Comment:
You do not need "Note that", it's implied by the "Note" heading that is automatically added by the `.. note::` directive